docs/src/api/class-androidwebview.md

```diff
@@ -21,3 +21,8 @@ WebView process PID.
 - returns: <[string]>
 
 WebView package identifier.
+
+## method: AndroidWebView.socketName
```

Comment:
Let's not expose it for now.

Reply:
OK. But check error: `Missing documentation for "AndroidWebView.socketName"`. How to handle it

Reply:
How to handle the lint error

Reply:
Call is `_socketName`